packages/material-ui/src/Tooltip/Tooltip.d.ts

```diff
@@ -19,7 +19,7 @@ export interface TooltipProps extends StandardProps<React.HTMLAttributes<HTMLDiv
   classes?: {
     /** Styles applied to the Popper component. */
     popper?: string;
-    /** Styles applied to the Popper component if `interactive={true}`. */
+    /** Styles applied to the Popper component unless `disableInteractive={true}`. */
```

Comment:
We have used the following pattern in the other components:

```suggestion
    /** Styles applied to the Popper component if `disableInteractive={false}`. */
```

Should we stick with it? cc @mbrookes

Reply:
The thing is that you'll almost never see `someBooleanProp={false}` with our API design philosophy. 

It's also incomplete since the styles are also applied without any code. My approach is covering `false` and `undefined` while your approach only covers `false`.

Reply:
I personally prefer "unless true" over "if false".